internal/eventingester/store/eventstore_test.go

```diff
@@ -0,0 +1,63 @@
+package store
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/G-Research/armada/internal/eventingester/model"
+
+	"github.com/go-redis/redis"
+
+	"github.com/G-Research/armada/internal/eventingester/configuration"
```

Comment:
Technically these imports are valid, but ideally they'd be 3 groups. However no tooling seems capable of enforcing this :(